inc/cleantalk-settings.php

```diff
@@ -1353,7 +1353,7 @@ function apbct_settings__error__output($return = false)
                     default:
                         $icon = '<span class="dashicons dashicons-hammer" style="color: red;"></span>';
                 }
-                $out .= '<h4>' . $icon . ' ' . Escape::escHtml($value) . '</h4>';
+                $out .= '<h4>' . $icon . ' ' . apbct_render_links_to_tag(Escape::escHtml($value)) . '</h4>';
```

Comment:
Probably it would be better to move escaping to the render function

Reply:
Your are right. Fixed it.